src/Packetery/Core/Validator/Order.php

```diff
@@ -18,6 +18,15 @@
  */
 class Order {
 
+	public const ERROR_NUMBER                     = 'number';
```

Comment:
Prosím přejmenovat konstanty tak, aby bylo zjevné že jde o klíč překladu. Je to důležité při sestavování pole překladů.